stripe/src/main/java/com/stripe/android/googlepay/StripeGooglePayContract.kt

```diff
@@ -128,3 +129,31 @@ internal class StripeGooglePayContract :
         }
     }
 }
+
+internal fun Status.getErrorMessage(): String? {
+    return when (this) {
+        Status.RESULT_SUCCESS -> null
+        Status.RESULT_INTERNAL_ERROR, Status.RESULT_CANCELED, Status.RESULT_DEAD_CLIENT -> "An internal error occurred."
+        else -> {
+            when (this.statusCode) {
```

Comment:
This is great. Should we move these to strings.xml?

Reply:
hehe, yeah, my comment on moving to strings got lost when I updated.